web/base_viewer.js

```diff
@@ -77,6 +77,7 @@ const DEFAULT_CACHE_SIZE = 10;
  *   total pixels, i.e. width * height. Use -1 for no limit. The default value
  *   is 4096 * 4096 (16 mega-pixels).
  * @property {IL10n} l10n - Localization service.
+ * @property {boolean} [enableScripting] - Enable JS evaluation.
```

Comment:
Please mention the default value here, and make sure to use the *same* description in both `BaseViewer` and `PDFPageView`, i.e.
```suggestion
 * @property {boolean} [enableScripting] - Enable embedded script execution.
 *   The default value is `false`.
```